### ISSUE TITLE
Question: Sealed Types => ScrollViewer, TextBlock, TickBar ?

### ISSUE THREAD

**Issue by VijayanRamachandran** [microsoft/microsoft-ui-xaml#3035]
Hi,

I am currently doing some prototypes migrating custom controls available in our end from WPF to WinUI.

During prototype I noticed that, in our end there are some WPF custom controls inheriting 
from ScrollViewer, TextBlock, TickBar ( available in System.Windows.Controls ) and when we try to migrate those custom controls to WinUI then ScrollViewer, TextBlock and TickBar control types ( available in Microsoft.UI.Xaml.Controls ) are sealed.

I hope making these types as selaed is just intermediate step from your end and in near future preview release those type will be removed from being sealed.

Is my understanding is correct or i am missing some point here ?

I am using right now
VS => Version 16.7.0 Preview 5.0
.Net SDK => dotnet-sdk-5.0.100-preview.4.20268.1-win-x64 / dotnet-sdk-5.0.100-preview.4.20268.1-win-x86

Thanks in advance.

**Comment by Felix-Dev** [comment]
Unsealing WinUI classes is on the mind of the team: https://github.com/microsoft/microsoft-ui-xaml/issues/780 It might take a while to arrive though.

**Comment by robloo** [comment]
This has been open a long time and honestly should be pretty quick to do. There is no reason this shouldn't be in the WinUI 3.0 previews.

**Comment by Felix-Dev** [comment]
@robloo It comes down to priorities. There are just so many things the community wants the WinUI team to solve that some things will unfortunately be put on the backlog for now. For example, parts of the community have been very vocal about host backdrop acrylic missing in WinUI 3.0. Or bringing UWP titlebar customization APIs to WinUI Desktop. 

Lats time I talked to @ryandemopoulos he said that this is not one of the most requested features so the focus lies on other things for now. He also said the team would like to tackle this but they are only so many folks with so much to do.

That said, the plans are changeable so the community should make itself heard if they want unsealed classes more than features like in-built titlebar customization, host-backdrop acrylic or better XAML Island technology.

**Comment by robloo** [comment]
@Felix-Dev Yes, I understand and you're right. As you said it just comes down to what features will be prioritized which unfortunately means whomever is loudest has a better chance of winning. This is a very simple change as I understand it though. It's something good to do to take your mind off harder problems :) That said, I don't need it personally so perhaps should quiet down for those features people do need.

**Comment by StephenLPeters** [comment]
@MikeHillberg FYI

**Comment by VijayanRamachandran** [comment]
@Felix-Dev @robloo @StephenLPeters 

As per the conversation i understand that it will take some time to unseal the classes. No worries. Let me wait for some more time till it is unsealed.

Thanks for the quick feedback.

**Comment by robloo** [comment]
@StephenLPeters Should this one now be closed in favor of #780? The question itself has been answered I think.
